tests/ops/integration_tests/saas/test_shippo_task.py

```diff
@@ -0,0 +1,116 @@
+import random
+import time
+
+import pytest
+import requests
```

Comment:
Make sure you get rid of the unused imports, the Pylance module in VS Code should help, we can go over the installation during our working sessions